2023-03-28 20:07:07,294 INFO    MainThread:14712 [wandb_setup.py:_flush():68] Configure stats pid to 14712
2023-03-28 20:07:07,294 INFO    MainThread:14712 [wandb_setup.py:_flush():68] Loading settings from C:\Users\andre\.config\wandb\settings
2023-03-28 20:07:07,294 INFO    MainThread:14712 [wandb_setup.py:_flush():68] Loading settings from D:\10.semester\NN\neural-networks\1.zadanie\src\wandb\settings
2023-03-28 20:07:07,294 INFO    MainThread:14712 [wandb_setup.py:_flush():68] Loading settings from environment variables: {'_require_service': 'True'}
2023-03-28 20:07:07,294 INFO    MainThread:14712 [wandb_setup.py:_flush():68] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2023-03-28 20:07:07,296 INFO    MainThread:14712 [wandb_init.py:_log_setup():492] Logging user logs to D:\10.semester\NN\neural-networks\1.zadanie\src\wandb\run-20230328_200707-hy5kvcb5\logs\debug.log
2023-03-28 20:07:07,296 INFO    MainThread:14712 [wandb_init.py:_log_setup():493] Logging internal logs to D:\10.semester\NN\neural-networks\1.zadanie\src\wandb\run-20230328_200707-hy5kvcb5\logs\debug-internal.log
2023-03-28 20:07:07,296 INFO    MainThread:14712 [wandb_init.py:init():532] calling init triggers
2023-03-28 20:07:07,297 INFO    MainThread:14712 [wandb_init.py:init():538] wandb.init called with sweep_config: {}
config: {}
2023-03-28 20:07:07,297 INFO    MainThread:14712 [wandb_init.py:init():554] re-initializing run, found existing run on stack: 4rf27ywd
2023-03-28 20:07:07,299 INFO    MainThread:14712 [wandb_run.py:_finish():1854] finishing run neural-netwok-nn/neural-network/4rf27ywd
2023-03-28 20:07:07,300 INFO    MainThread:14712 [jupyter.py:save_history():449] not saving jupyter history
2023-03-28 20:07:07,300 INFO    MainThread:14712 [jupyter.py:save_ipynb():379] not saving jupyter notebook
2023-03-28 20:07:07,300 INFO    MainThread:14712 [wandb_init.py:_jupyter_teardown():420] cleaning up jupyter logic
2023-03-28 20:07:07,300 INFO    MainThread:14712 [wandb_run.py:_atexit_cleanup():2083] got exitcode: 0
2023-03-28 20:07:07,301 INFO    MainThread:14712 [wandb_run.py:_restore():2066] restore
2023-03-28 20:07:07,301 INFO    MainThread:14712 [wandb_run.py:_restore():2072] restore done
2023-03-28 20:07:13,956 INFO    MainThread:14712 [wandb_run.py:_footer_history_summary_info():3442] rendering history
2023-03-28 20:07:13,956 INFO    MainThread:14712 [wandb_run.py:_footer_history_summary_info():3474] rendering summary
2023-03-28 20:07:13,961 INFO    MainThread:14712 [wandb_run.py:_footer_sync_info():3398] logging synced files
2023-03-28 20:07:13,968 INFO    MainThread:14712 [wandb_init.py:init():588] starting backend
2023-03-28 20:07:13,968 INFO    MainThread:14712 [wandb_init.py:init():592] setting up manager
2023-03-28 20:07:13,972 INFO    MainThread:14712 [backend.py:_multiprocessing_setup():106] multiprocessing start_methods=spawn, using: spawn
2023-03-28 20:07:13,975 INFO    MainThread:14712 [wandb_init.py:init():599] backend started and connected
2023-03-28 20:07:13,983 INFO    MainThread:14712 [wandb_run.py:_label_probe_notebook():1203] probe notebook
2023-03-28 20:07:13,984 INFO    MainThread:14712 [wandb_run.py:_label_probe_notebook():1213] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2023-03-28 20:07:13,984 INFO    MainThread:14712 [wandb_init.py:init():687] updated telemetry
2023-03-28 20:07:14,037 INFO    MainThread:14712 [wandb_init.py:init():727] communicating run to backend with 60.0 second timeout
2023-03-28 20:07:15,425 INFO    MainThread:14712 [wandb_run.py:_on_init():2134] communicating current version
2023-03-28 20:07:15,506 INFO    MainThread:14712 [wandb_run.py:_on_init():2143] got version response upgrade_message: "wandb version 0.14.0 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2023-03-28 20:07:15,506 INFO    MainThread:14712 [wandb_init.py:init():775] starting run threads in backend
2023-03-28 20:07:15,684 INFO    MainThread:14712 [wandb_run.py:_console_start():2114] atexit reg
2023-03-28 20:07:15,684 INFO    MainThread:14712 [wandb_run.py:_redirect():1969] redirect: SettingsConsole.WRAP_RAW
2023-03-28 20:07:15,684 INFO    MainThread:14712 [wandb_run.py:_redirect():2034] Wrapping output streams.
2023-03-28 20:07:15,686 INFO    MainThread:14712 [wandb_run.py:_redirect():2059] Redirects installed.
2023-03-28 20:07:15,686 INFO    MainThread:14712 [wandb_init.py:init():817] run started, returning control to user process
2023-03-28 20:07:15,686 INFO    MainThread:14712 [wandb_config.py:__setitem__():155] config set learning_rate = 0.001 - <bound method Run._config_callback of <wandb.sdk.wandb_run.Run object at 0x00000163C18965F0>>
2023-03-28 20:07:15,687 INFO    MainThread:14712 [wandb_run.py:_config_callback():1250] config_cb learning_rate 0.001 None
2023-03-28 20:07:15,687 INFO    MainThread:14712 [wandb_config.py:__setitem__():155] config set batch_size = 32 - <bound method Run._config_callback of <wandb.sdk.wandb_run.Run object at 0x00000163C18965F0>>
2023-03-28 20:07:15,687 INFO    MainThread:14712 [wandb_run.py:_config_callback():1250] config_cb batch_size 32 None
2023-03-28 20:07:15,687 INFO    MainThread:14712 [wandb_config.py:__setitem__():155] config set epochs = 50 - <bound method Run._config_callback of <wandb.sdk.wandb_run.Run object at 0x00000163C18965F0>>
2023-03-28 20:07:15,687 INFO    MainThread:14712 [wandb_run.py:_config_callback():1250] config_cb epochs 50 None
2023-03-28 20:08:27,847 INFO    MainThread:14712 [wandb_setup.py:_flush():68] Configure stats pid to 14712
2023-03-28 20:08:27,847 INFO    MainThread:14712 [wandb_setup.py:_flush():68] Loading settings from C:\Users\andre\.config\wandb\settings
2023-03-28 20:08:27,848 INFO    MainThread:14712 [wandb_setup.py:_flush():68] Loading settings from D:\10.semester\NN\neural-networks\1.zadanie\src\wandb\settings
2023-03-28 20:08:27,848 INFO    MainThread:14712 [wandb_setup.py:_flush():68] Loading settings from environment variables: {'_require_service': 'True'}
2023-03-28 20:08:27,848 INFO    MainThread:14712 [wandb_setup.py:_flush():68] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2023-03-28 20:08:27,848 INFO    MainThread:14712 [wandb_init.py:_log_setup():492] Logging user logs to D:\10.semester\NN\neural-networks\1.zadanie\src\wandb\run-20230328_200827-g1m1lo2t\logs\debug.log
2023-03-28 20:08:27,848 INFO    MainThread:14712 [wandb_init.py:_log_setup():493] Logging internal logs to D:\10.semester\NN\neural-networks\1.zadanie\src\wandb\run-20230328_200827-g1m1lo2t\logs\debug-internal.log
2023-03-28 20:08:27,849 INFO    MainThread:14712 [wandb_init.py:_jupyter_setup():438] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x00000163C0FE6A70>
2023-03-28 20:08:27,849 INFO    MainThread:14712 [wandb_init.py:init():532] calling init triggers
2023-03-28 20:08:27,849 INFO    MainThread:14712 [wandb_init.py:init():538] wandb.init called with sweep_config: {}
config: {}
2023-03-28 20:08:27,849 INFO    MainThread:14712 [wandb_init.py:init():554] re-initializing run, found existing run on stack: hy5kvcb5
2023-03-28 20:08:27,851 INFO    MainThread:14712 [wandb_run.py:_finish():1854] finishing run neural-netwok-nn/neural-network/hy5kvcb5
2023-03-28 20:08:27,851 INFO    MainThread:14712 [wandb_run.py:_atexit_cleanup():2083] got exitcode: 0
2023-03-28 20:08:27,852 INFO    MainThread:14712 [wandb_run.py:_restore():2066] restore
2023-03-28 20:08:27,852 INFO    MainThread:14712 [wandb_run.py:_restore():2072] restore done
2023-03-28 20:08:35,198 INFO    MainThread:14712 [wandb_run.py:_footer_history_summary_info():3442] rendering history
2023-03-28 20:08:35,199 INFO    MainThread:14712 [wandb_run.py:_footer_history_summary_info():3474] rendering summary
2023-03-28 20:08:35,209 INFO    MainThread:14712 [wandb_run.py:_footer_sync_info():3398] logging synced files
